src/@next/PasswordInput/PasswordInputStyle.ts

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+export const StyledSuffixContainer = styled.div`
+  height: 17px;
```

Comment:
@michac789 size of a suffix icon should be 20px x 20px with 8px of paddings on the left and right
![image](https://github.com/glints-dev/glints-aries/assets/12844836/fedc0022-d432-4eb7-a1d9-7f7563162e72)